keyboards/clawsome/numeros/keymaps/default/keymap.c

```diff
@@ -0,0 +1,41 @@
+/* Copyright 2020 AAClawson (AlisGraveNil)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+   /*
+    * ---------------------
+    * |NUM | /  | *  | -  |
+    * ---------------------
+    * | 7  | 8  | 9  | +  |
+    * ---------------|    |
+    * | 4  | 5  | 6  |    |
+    * ---------------------
+    * | 1  | 2  | 3  |ENT |
+    * ---------------|    |
+    * | 0       | .  |    |
+    * ---------------------
+    */
+    [0] = LAYOUT(
+        KC_NLCK, KC_PSLS, KC_PAST, KC_PMNS,
+        KC_P7,   KC_P8,   KC_P9,   KC_PPLS,
+        KC_P4,   KC_P5,   KC_P6,
+        KC_P1,   KC_P2,   KC_P3,   KC_PENT,
+        KC_P0,            KC_PDOT
```

Comment:
```suggestion
        KC_P7,   KC_P8,   KC_P9,
        KC_P4,   KC_P5,   KC_P6,   KC_PPLS,
        KC_P1,   KC_P2,   KC_P3,
        KC_P0,            KC_PDOT, KC_PENT,
```

Reply:
<img width="318" alt="Screen Shot 2020-07-16 at 07 38 26" src="https://user-images.githubusercontent.com/63020540/87677978-9d099100-c737-11ea-90b1-57351fb14e23.png">

With the wiring as showing in that snippet, will moving those labels still work, or will it confuse the QMK software?

Reply:
Yes, it will still work. The bottom half of the layout macro is what defines the electrical matrix.